sdk/src/main/java/com/kin/ecosystem/Kin.java

```diff
@@ -424,12 +424,10 @@ private static void checkInstanceNotNull() throws ClientException {
 	 */
 	public static void logout() throws ClientException {
 		checkInstanceNotNull();
-		if (isAccountLoggedIn.compareAndSet(true, false)) {
```

Comment:
you should keep `isAccountLoggedIn.compareAndSet(true, false)`.

Reply:
Thanks, Updated.